include/jemalloc/internal/mutex_types.h

```diff
@@ -7,25 +7,25 @@ typedef struct malloc_mutex_s malloc_mutex_t;
 #  define MALLOC_MUTEX_INITIALIZER
 #elif (defined(JEMALLOC_OS_UNFAIR_LOCK))
 #  define MALLOC_MUTEX_INITIALIZER					\
-     {OS_UNFAIR_LOCK_INIT, WITNESS_INITIALIZER("mutex", WITNESS_RANK_OMIT)}
+     {{{OS_UNFAIR_LOCK_INIT}}, WITNESS_INITIALIZER("mutex", WITNESS_RANK_OMIT)}
```

Comment:
Not sure I'm following. If (!DEBUG), WITNESS_INITIALIZER is empty (has a ifdef around it) so that we only initialize once for the first member (lock); if (DEBUG), then the braces and declaration order should ensure we initialize properly. When testing locally, gcc is only happy with the two braces around {{..._INIT}}, which I believe avoids ambiguity.

The static initializer is already used here: https://github.com/jemalloc/jemalloc/blob/dev/src/jemalloc.c#L236